### PR TITLE
add proxy_get code missed in rebase

### PR DESCRIPTION
**Pull request by bookshelfdave**
I diffed dip_pg14 & master and I believe everything is accounted for. This PR is **_REQUIRED**_ for 1.4

### CODE REVIEW

**Comment by bookshelfdave** [comment]
Additionally, I prevent erlang:registered() from being called on an already registered name. This was found as the result of a tweak made to the test (recommended by Andrew): https://github.com/basho/riak_test/pull/273

File: src/riak_core_cluster_mgr.erl [hunk 1]
```diff
@@ -223,12 +223,16 @@ init(Defaults) ->
...
+    State = register_defaults(Defaults, #state{
+                is_leader = false,
+                balancer_fun = BalancerFun,
+                cluster_id=ClusterID}),
 
     %% Schedule a delayed connection to know clusters
     schedule_cluster_connections(),
-
-    case riak_repl2_leader:leader_node() of
+   case riak_repl2_leader:leader_node() of
```
Comment (Vagabond) [inline src/riak_core_cluster_mgr.erl]: You hosed up the indentation on this line.

File: src/riak_repl2_pg_block_requester.erl [hunk 2]
```diff
@@ -158,9 +158,17 @@ handle_info({Proto, Socket, Data},
                 RemoteClusterID = list_to_binary(io_lib:format("~p",[ClusterID])),
                 lager:debug("Remote cluster id = ~p", [RemoteClusterID]),
                 lager:debug("Remote cluster name = ~p", [RemoteClusterName]),
-        RegName = riak_repl_util:make_pg_name(RemoteClusterName),
-        lager:debug("RegName = ~p",[RegName]),
-        erlang:register(RegName, self()),
+                RegName = riak_repl_util:make_pg_name(RemoteClusterName),
+                lager:debug("RegName = ~p",[RegName]),
+                case lists:member(RegName, erlang:registered()) of
```
Comment (Vagabond) [inline src/riak_repl2_pg_block_requester.erl]: Can you provide the rationale here? I don't really like this approach. Why would a process be still alive and registered if we no longer needed it?

Comment (bookshelfdave) [inline src/riak_repl2_pg_block_requester.erl]: My bad, I just fixed the symptom that was uncovered w/ riak_test.

**Comment by bookshelfdave** [comment]
btw, the two basic riak_tests:

```
 test_basic_pg_mode_repl13
 test_basic_pg_mode_mixed
```

would not run without the following change: https://github.com/basho/riak_repl/pull/285/files#L0R231

**Pull request closed without merging**